### PR TITLE
Feature/localisation

### PR DESCRIPTION
**Pull request by I-Valchev**

### CODE REVIEW

**Comment by I-Valchev** [comment]
Fixes #832 
Also this temporarily removes collections (due to issue with sets inside collections, revisit with #833 )

File: phpstan.neon [hunk 1]
```diff
@@ -17,6 +17,10 @@ parameters:
         -
             message: '#Unreachable statement - code above always terminates#'
             path: %currentWorkingDirectory%/src/*
+        -
```
Comment (bobdenotter) [inline phpstan.neon]: Add comment

File: src/Entity/Field.php [hunk 2]
```diff
@@ -173,19 +162,17 @@ public function setName(string $name): self
...
+    /**
+     * @Groups("get_field")
+     */
     public function getValue(): ?array
     {
-        return $this->value;
+        return $this->translate($this->getCurrentLocale())->getValue();
     }
 
-    /**
```
Comment (bobdenotter) [inline src/Entity/Field.php]: Place back comment

File: src/Entity/FieldTranslation.php [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="json")
+     * @Groups("get_field")
```
Comment (bobdenotter) [inline src/Entity/FieldTranslation.php]: remove

File: src/Entity/FieldTranslation.php [hunk 4]
```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Knp\DoctrineBehaviors\Contract\Entity\TranslationInterface;
+use Knp\DoctrineBehaviors\Model\Translatable\TranslationTrait;
+use Symfony\Component\Serializer\Annotation\Groups;
```
Comment (bobdenotter) [inline src/Entity/FieldTranslation.php]: remove

File: tests/behat/edit_record_1.feature [hunk 5]
```diff
@@ -204,69 +204,3 @@ Feature: Edit record
     Then I should be on "/bolt/edit/43#sets"
     And the field with css "#sets > div > div:nth-child(2) > div > input" should contain "Foo"
     And the field with css "#sets > div > div:nth-child(3) > div > textarea" should contain "Bar"
-
-  @javascript
```
Comment (bobdenotter) [inline tests/behat/edit_record_1.feature]: `@javascript-disabled`

**Comment by I-Valchev** [comment]
Fixes #832

**Comment by bobdenotter** [comment]
Looks much better! Thanks @I-Valchev 

And thank you to @TomasVotruba for your work on https://github.com/KnpLabs/DoctrineBehaviors 👍

**Comment by TomasVotruba** [comment]
Thanks for reaching out :)

**Pull request merged**